CSOAR-590 Align CSOAR API documentation (#3627)
SumoLogic/sumologic-documentation: docs/api/cloud-soar.md

<img src={useBaseUrl('img/icons/security/SOC.png')} alt="icon" width="60"/>

The Cloud SOAR APIs allow you to manage incidents, triage, and other Cloud SOAR features.

:::info

> If your instance URL is a subdomain of `soar.sumologic.com`, please refer to the 
> [next paragraph](#inline-api-documentation) for information about API.

Reviewer: ```suggestion
If your instance URL is a subdomain of `soar.sumologic.com`, refer to the [next section](#inline-api-documentation) for information about API.
```

<CloudSoarApi/>

## Inline API documentation

> For organizations having Cloud SOAR available at URLs matching the pattern `*.soar.sumologic.com`, 
> the API documentation is available through your Cloud SOAR instance at the following URL:

Reviewer: ```suggestion
For organizations having Cloud SOAR available at URLs matching the pattern `*.soar.sumologic.com`,  the API documentation is available through your Cloud SOAR instance at the following URL:
```

```
https://<cloudsoarhost>/incmansuite_ng/lib/gui/app.php#support_apidoc|api_documentation_v3
```

> For more information, see [Cloud SOAR APIs](/docs/cloud-soar/cloud-soar-apis/) in
> our [Cloud SOAR documentation](/docs/cloud-soar/).

Reviewer: ```suggestion
For more information, see [Cloud SOAR APIs](/docs/cloud-soar/cloud-soar-apis/).
```
